code/modules/clothing/suits/utility.dm

```diff
@@ -130,3 +130,38 @@
 /obj/item/clothing/suit/radiation/Initialize()
 	. = ..()
 	slowdown_per_slot[slot_shoes] = 1.5
+
+/*
+ * chemical protection
+ */
+
+/obj/item/clothing/head/chem_hood
+	name = "chemical hood"
+	icon_state = "chem_hood"
+	desc = "A hood that protects the head from chemical comtaminants."
+	permeability_coefficient = 0
+	armor = list(
+		bio = ARMOR_BIO_RESISTANT, 
+		rad = ARMOR_RAD_MINOR
+		)
+	flags_inv = HIDEEARS|HIDEEYES|BLOCKHAIR
+	item_flags = ITEM_FLAG_THICKMATERIAL
+	body_parts_covered = HEAD
+	siemens_coefficient = 0.9
```

Comment:
If it's intended to be identical in stats to a bio suit, why not just make it `/obj/item/clothing/head/bio_hood/chem`?

Reply:
has separate stats. lowered the bio protection, and doesn't cover face and eyes.